CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [2.x-dev]
+
+* SilverStripe 4.0 support.
+
+## [1.x-dev]
```

Comment:
Personally I'm not a fan of having changelogs in the code - they get forgotten about fairly often. I prefer putting release notes in the GitHub releases instead, but just my two cents

Reply:
Discussed - we'll discuss this sort of thing another time